test/extended/cli/basics.go

```diff
@@ -0,0 +1,121 @@
+		o.Expect(out).To(o.ContainSubstring("a a a a"))
+
+		out, err = oc.Run("get").Args("-f", file, `--output=jsonpath="{..metadata.annotations.myannotation}"`).Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("b b b b"))
+
+		err = oc.Run("delete").Args("-f", file).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+	})
+
+	g.It("can create deploymentconfig and clusterquota", func() {
+		nginx := k8simage.GetE2EImage(k8simage.Nginx)
+
+		err := oc.Run("create").Args("dc", "my-nginx", "--image="+nginx).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = oc.Run("delete").Args("dc", "my-nginx").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// need admin here
+		ocAdmin := oc.AsAdmin()
+		err = ocAdmin.Run("create").Args("clusterquota", "limit-bob", "--project-label-selector=openshift.io/requestor=user-bob", "--hard=pods=10").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = ocAdmin.Run("delete").Args("clusterquota", "limit-bob").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+	})
+
+	g.It("can patch resources", func() {
+		// need admin here
+		ocAdmin := oc.AsAdmin()
+
+		err := ocAdmin.Run("adm").Args("groups", "new", "patch-group").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		err = ocAdmin.Run("patch").Args("group", "patch-group", "-p", `users: ["myuser"]`, "--loglevel=8").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		out, err := ocAdmin.Run("get").Args("group", "patch-group", "-o", "yaml").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("myuser"))
+
+		err = ocAdmin.Run("patch").Args("group", "patch-group", "-p", `users: []`, "--loglevel=8").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		// apply same patch twice results in "not patched" text
+		out, err = ocAdmin.Run("patch").Args("group", "patch-group", "-p", `users: []`).Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring(`patched (no change)`))
+
+		// applying an invalid patch results in exit code 1 with error text
+		out, err = ocAdmin.Run("patch").Args("group", "patch-group", "-p", `users: ""`).Output()
+		o.Expect(err).To(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring(`cannot restore slice from string`))
+
+		out, err = ocAdmin.Run("get").Args("group", "patch-group", "-o", "yaml").Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(out).To(o.ContainSubstring("users: []"))
+
+		err = ocAdmin.Run("delete").Args("group", "patch-group").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+	})
+
+	g.It("can describe an OAuth access token", func() {
+		// need admin here
+		ocAdmin := oc.AsAdmin()
+
+		err := ocAdmin.Run("create").Args("-f", oauthAccessTokenFile).Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		out, err := ocAdmin.Run("describe").Args("oauthaccesstoken", "sha256~efaca6fab897953ffcb4f857eb5cbf2cf3a4c33f1314b4922656303426b1cfc9").Output()
```

Comment:
Nit: you forgot to drop that from basicresource.sh here: https://github.com/openshift/origin/blob/e000efe2de764b8f0789bbea7bc2ebe248eb5275/test/extended/testdata/cmd/test/cmd/basicresources.sh#L198-L201